deployment/ansible/otp.yml

```diff
@@ -7,5 +7,4 @@
       apt: update_cache=yes
 
   roles:
-    - { role: "cac-tripplanner.otp"}
```

Comment:
I understand azavea.opentripplanner will be run by dependency, but maybe it makes sense to define it here explicitly too since at the end of the day we want this VM to run the OTP server.